recipes/paleomix/LICENSE

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
You should not need to add this. (It will be retrieved from the downloaded source tarball.) Also, something is wrong: This is not the GPLv3.

Reply:
circleci was complaining about missing LICENSE file... On other recipes I did not have the issue though (the LICENSE file was pulled directly from the github repo).

Reply:
Yes, you‘re right. Downloading and unpacking the paleomix-1.2.14.tar.gz file manually, one can see that there are actually two license files `licenses/gpl.txt` and `licenses/mit.txt`. I don’t know if `license_file` supports multiple license files, but if it does, then you could refer to those files.